mmv1/templates/terraform/examples/compute_storage_pool_basic.tf.tmpl

```diff
@@ -0,0 +1,13 @@
+resource "google_compute_storage_pool" "{{$.PrimaryResourceId}}" {
+  name = "{{index $.Vars "storage_pool_name"}}"
+
+  pool_provisioned_capacity_gb = "10240"
+
+  pool_provisioned_throughput = 100
+
+  storage_pool_type = "https://www.googleapis.com/compute/v1/projects/${data.google_project.project.project_id}/zones/us-central1-a/storagePoolTypes/hyperdisk-throughput"
```

Comment:
```suggestion
  storage_pool_type = "https://www.googleapis.com/compute/beta/projects/${data.google_project.project.project_id}/zones/us-central1-a/storagePoolTypes/hyperdisk-throughput"
```

It appears that v1 is being sent in the api request for `storage_pool_type` but beta is returned.

this field may need to be version dependent? I am not sure if there is president for it. If we can get the storage pool type from a datasource that could be a solution.

Reply:
Adding storagePoolTypes datsource seems reasonable way. I'll add it.

Reply:
can you tag me on the change. I'll prioritize review